admin-dev/themes/default/webpack.config.js

```diff
@@ -0,0 +1,46 @@
+var path = require('path');
+var webpack = require('webpack');
+
+module.exports = {
+  entry: [
+    'cldrjs/dist/cldr.js',
```

Comment:
Would probably be easier to maintain if we have a single entry point, say `admin-dev/themes/default/theme.js` that imports all of the deps